package-lock.json

```diff
@@ -1830,6 +1830,8 @@
     },
     "@testing-library/jest-dom": {
       "version": "5.11.10",
+      "resolved": "https://registry.npmjs.org/@testing-library/jest-dom/-/jest-dom-5.11.10.tgz",
```

Comment:
not sure exactly what's going on here. But if nothing broke and things are looking great-- good job!